mayastor/src/bin/mayastor-client/nexus_cli.rs

```diff
@@ -321,6 +332,40 @@ async fn nexus_unpublish(
     Ok(())
 }
 
+async fn nexus_set_nvme_ana_state(
+    mut ctx: Context,
+    matches: &ArgMatches<'_>,
+) -> Result<(), Status> {
+    let uuid = matches.value_of("uuid").unwrap().to_string();
+    let ana_state = match matches.value_of("ana_state").unwrap() {
```

Comment:
Could we add a `impl FromStr for rpc::NvmeAnaState` somewhere in the codebase and use `.parse()` instead?

Reply:
Done, in the rpc crate.